src/state/effects/helpers.ts

```diff
@@ -17,11 +17,13 @@ export function isVisibleEffect<T extends Effect>(e: T): e is VisibleEffect<T> {
 export function isCapacityEffect<T extends Effect>(
   e: T,
 ): e is CapacityEffect<T> {
-  return Object.getOwnPropertyNames(e).includes('capacity');
+  const key: keyof CapacityEffect<T> = 'capacityChange';
+  return Object.getOwnPropertyNames(e).includes(key);
```

Comment:
the typeguard implementation also was not changed by vscode refactor. explicitly using `keyof` should make the issue easier to find